homework/D_Miftari_Assignment_1/DMiftari_DW_homework.sql

```diff
@@ -0,0 +1,80 @@
+DROP DATABASE dw_database_assignment;
+CREATE DATABASE dw_database_assignment;
+\c dw_database_assignment;
+
+CREATE TABLE offices (
+office_code INTEGER PRIMARY KEY,
+city VARCHAR,
+state VARCHAR,
+country VARCHAR,
+office_location VARCHAR
+);
+
+--offices = employees[['office_code', 'city', 'state', 'country', 'office_location']].copy()
```

Comment:
?? Remove commented code! Keep it in commits if you want to return to it. But why would you want to return to python code in a sql file??